runner/javaut.go

```diff
@@ -99,15 +106,13 @@ func JUnit(ctx context.Context, tests, code io.Reader) (*model.JunitTestResult,
 		if err := d.Decode(&utr); err != nil {
 			return nil, err
 		}
-		testResults = &model.JunitTestResult{
-			Stdout:   stdout.String(),
-			Results:  utr,
-			Stderr:   stderr.String(),
-			Start:    start,
-			End:      end,
-			Endpoint: "junit-result",
-		}
+		testResults.Results = utr
 	}
 
-	return testResults, <-errc
+	derr := <-errc
+	if derr != nil && stderr.String() == "" {
+		//Tests are missing.
+		testResults.Stderr = "There are no tests to run."
```

Comment:
You are setting standard error tome some arbitrary string that did not even come from inside the container. That's not gonna happen.